frontend/public/components/chargeback.tsx

```diff
@@ -1,682 +0,0 @@
-import * as React from 'react';
```

Comment:
Please remove any associated models, YAML templates, and feature flags (`CHARGEBACK`, `CAN_LIST_CHARGEBACK_REPORTS`) for chargeback.

https://github.com/spadgett/console/blob/5ed583c6ecb31fa6b4f1ad4dbf771cdc6a20aa27/frontend/public/models/yaml-templates.ts#L196-L227

https://github.com/spadgett/console/blob/5ed583c6ecb31fa6b4f1ad4dbf771cdc6a20aa27/frontend/public/models/index.ts#L54-L84